src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = {};
+
+  sourceString
+    .trim()
+    .split(';')
+    .forEach((element) => {
+      const [key, value] = element.split(':');
+
+      if (value !== undefined) {
+        obj[key.trim()] = value.trim().replace(/\\n/g, '\n');
+      }
```

Comment:
Instead of using forEach which doesn't return a new array, consider using map or reduce which are more suitable for transforming an array into an object.